alpha/hal.cpp

```diff
@@ -304,6 +304,30 @@ void _TS_HAL::power_set_mic(bool enabled)
   EXIT_CRITICAL;
 }
 
+void _TS_HAL::power_set_charging_current(uint8_t current) {
```

Comment:
we can save on functions and CS if we put these settings in hal init instead of alpha.ino

these are set once on startup and probably don't need wrappers